packages/leafygreen-provider/src/OverlayContext/OverlayContext.spec.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { createRef, useState } from 'react';
+import { fireEvent, render } from '@testing-library/react';
```

Comment:
Since we're only firing click events in our tests, we may want to consider using `userEvent` instead of `fireEvent` to better replicate actual user interaction. A bit more reasoning here: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-testing-libraryuser-event